oauth2/oauth2-core/src/test/java/org/springframework/security/oauth2/core/web/reactive/function/OAuth2BodyExtractorsTests.java

```diff
@@ -120,4 +120,33 @@ public void oauth2AccessTokenResponseWhenValidThenCreated() throws Exception {
 		assertThat(result.getRefreshToken().getTokenValue()).isEqualTo("tGzv3JOkF0XG5Qx2TlKWIA");
 		assertThat(result.getAdditionalParameters()).containsEntry("example_parameter", "example_value");
 	}
+
+
+	@Test
+	// Fixes: gh-6087
```

Comment:
While the convention is that the commit message says "Fixes: ", the comment should just be gh-6087. Can you please change this and then I think we are ready to merge.

Reply:
yup